Cookbook/Technical-Documents/DebuggingPushNotifications.md

```diff
@@ -0,0 +1,37 @@
+# Debugging Push Notifications
+
+To sucessfully debug Push Notification on your device make sure that you have proper configurations set up as described in this document.
+
+- [PubNub](#pubnub)
```

Comment:
Nit  - TL;DR tends to be on the top :)

Reply:
Then no one will read the doc! =)

Reply:
And they should read the doc, even if they just want TL;DR? ;) 

In the end we tend to have summary :)

Anyway, I think the document needs a clearer structure

Reply:
of course they should read the doc, otherwise I just wrote it for myself =P
I've added more sections as you suggested